include/logger/loggerApi.h

```diff
@@ -25,8 +25,12 @@
 
 #define LOGGER_API \
```

Comment:
AFAIK #defines are symbols for the compiler to use, they don't go into the data section.